code/modules/clothing/clothing.dm

```diff
@@ -987,6 +987,45 @@
 		to_chat(usr, "<span class='notice'>You roll down your [src]'s sleeves.</span>")
 	update_clothing_icon()
 
+//Switch to taur sprites if a taur equips
+/obj/item/clothing/suit
+	var/taurized = FALSE
+
+/obj/item/clothing/suit/equipped(var/mob/user, var/slot)
+	var/normalize = TRUE
+
+	//Pyramid of doom-y. Improve somehow?
+	if(!taurized && slot == slot_wear_suit && ishuman(user))
+		var/mob/living/carbon/human/H = user
+		if(isTaurTail(H.tail_style))
+			var/datum/sprite_accessory/tail/taur/taurtail = H.tail_style
+			if(taurtail.suit_sprites && (get_worn_icon_state(slot_wear_suit_str) in cached_icon_states(taurtail.suit_sprites)))
+				icon_override = taurtail.suit_sprites
+				normalize = FALSE
+				taurized = TRUE
+
+	if(normalize && taurized)

```

Comment:
if a taur equips this, and then another taur equips it (consecutively), it will return to the non-taur sprite for the second taur. `normalize` becomes useless with the above suggested changes to the nested ifs